draft-ietf-sidrops-aspa-verification.xml

```diff
@@ -310,26 +303,102 @@ authorized(AS x, AS y) =  / Else, "Provider+" if the VAP-SPAS entry
         If an attacker creates a route leak intentionally, they may try to strip their AS from the AS_PATH.
         To partly guard against that, a check is necessary to match the most recently added AS in the AS_PATH to the BGP neighbor's ASN.
         This check MUST be performed as specified in Section 6.3 of <xref target="RFC4271"/>.
-        If the check fails, then the AS_PATH is considered a Malformed AS_PATH and the UPDATE is considered to be in error (Section 6.3 of <xref target="RFC4271"/>).
+        If the check fails, then the AS_PATH is considered a Malformed AS_PATH and the UPDATE is in error (Section 6.3 of <xref target="RFC4271"/>).
```

Comment:
TYPO?

Reply:
@mitradir 
I have restored the sentence back to how it was: 
"If the check fails, then the AS_PATH is
   considered a Malformed AS_PATH and the UPDATE is considered to be in
   error (Section 6.3 of [RFC4271])."
Let me know if you meant something else by "TYPO". This sentence is borrowed from RFC 4271.